apps/scene2pset/scene2pset.cc

```diff
@@ -104,13 +173,13 @@ main (int argc, char** argv)
     args.add_option('p', "poisson-normals", false, "Scale normals according to confidence");
     args.add_option('S', "scale-factor", true, "Factor for computing scale values [2.5]");
     args.add_option('F', "fssr", true, "FSSR output, sets -nsc and -di with scale ARG");
+    args.add_option('C', "correspondence", false, "Output correspondence data (only in the absence of -m and -b)");
```

Comment:
Keep lines to 80 chars.
`Output correspondences (in absence of -m and -b only)`